paddlenlp/transformers/qwen/modeling.py

```diff
@@ -118,23 +137,55 @@ def __init__(self, config):
 
         self.attn_dropout = nn.Dropout(config.attn_dropout_prob)
 
-    def _attn(self, query, key, value, attention_mask=None):
-        attn_weights = paddle.matmul(query, key.transpose([0, 1, 3, 2]))
-
-        if self.scale_attn_weights:
-            attn_weights = attn_weights * self.inv_norm_factor
-
-        attn_weights = attn_weights + attention_mask
-
-        attn_weights = nn.functional.softmax(attn_weights, axis=-1)
-
-        attn_weights = attn_weights.astype(value.dtype)
-        attn_weights = self.attn_dropout(attn_weights)
-
-        attn_output = paddle.matmul(attn_weights, value)
-        attn_output = attn_output.transpose([0, 2, 1, 3])
-
-        return attn_output, attn_weights
+    def _attn(self, query, key, value, config, attention_mask=None):
+        # Support the flash attention and normal attention
+        bsz, q_len, num_heads, head_dim = query.shape
+        _, kv_seq_len, _, _ = value.shape
+
+        if config.use_flash_attn and flash_attention is not None:
```

Comment:
```suggestion
        if config.use_flash_attention and flash_attention is not None:
```
参数名统一用 `use_flash_attention`

Reply:
这里是因为config文件写死了，需要后续统一修改config文件